src_c/mixer.c

```diff
@@ -261,18 +261,45 @@ _format_view_to_audio(Py_buffer *view)
     return format;
 }
 
+static void
+_pg_push_mixer_event(int type, int code)
+{
+    pgEventObject *e;
+    PyObject *dict, *dictcode;
+    SDL_Event event;
+    PyGILState_STATE gstate = PyGILState_Ensure();
+
+    dict = PyDict_New();
```

Comment:
There's no error checking.

Reply:
It is checking errors, but failing silently. We can’t do anything about errors from callback functions

Reply:
There's two ways to handle tracebacks in callbacks. 
- 1) log the traceback in the handler
- 2) note the traceback and then raise the traceback somewhere else, perhaps in event get/pump/wait.

Two is more the python way.

Reply:
Even the code that was there previously did not do any error handling. It’s much more work to handle those errors. Much easier to fail silently (maybe I’m just being lazy here ;) )

Reply:
Asyncio does the first, and since then (in hindsight) it has become the consensus that this was a mistake, and async event handling should probably do the second.